README.md

```diff
@@ -125,3 +125,23 @@ Adds item to html report as link:
 
 tool.htmlReporter.addExtraItem(linkText, linkUrl)
 ```
+
+### addMetaInfoExtender
+
+Extend meta-info of each test using passed data:
+
+```js
+tool.htmlReporter.addMetaInfoExtender(name, value);
+```
+
+* **name** (required) `String` - name of meta info
+* **value** (required) `String|Number|Function` - value of meta info. If a function is passed, then `suite` and `extraItems` will be passed to its arguments.
```

Comment:
It is unclear what is inside `extraItems`

Reply:
okay, I will add an example